frontend/src/pages/ClinicalRecordsPage.tsx

```diff
@@ -3,11 +3,15 @@ import ClinicalRecords from '../components/ClinicalRecords/ClinicalRecords';
 export default function ClinicalRecordsPage() {
   return (
     <section className="flex flex-col gap-2 p-3 md:p-10">
```

Comment:
Cuando se usa flex-col por defecto no viene centrado
```suggestion
    <section className="flex flex-col items-center gap-2 p-3 md:p-10">
```